secboot/secboot.go

```diff
@@ -34,6 +34,17 @@ import (
 )
 
 const (
+	// The range 0x01880005-0x0188000F
+	//
+	// FIXME: we should apply for a subrange from UAPI group once
+	// they got a range assigned by TCG.  See
+	// https://github.com/uapi-group/specifications/pull/118
+	// For now we use a sub range on the unassigned owner handles
+	PCRPolicyCounterHandleStart = uint32(0x01880005)
+	PCRPolicyCounterHandleRange = uint32(0x01880010 - 0x01880005)
```

Comment:
We might want to move this definition to secboot at some point. I have a need to transiently create a NV index as part of rotating the authorization value for the lockout hierarchy.